docs/guides/expectations/create_expectations_overview.md

```diff
@@ -3,4 +3,67 @@ title: "Create Expectations: Overview"
 ---
 # [![Create Expectations Icon](../../images/universal_map/Flask-active.png)](./create_expectations_overview.md) Create Expectations: Overview
 
-** This is a placeholder until content is created, so that corresponding links don't cause docusaurus to break **
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='inactive'/>
+
+:::note Prerequisites
+- Completing [Step 3: Create Expectations](/docs/tutorials/getting_started/create_your_first_expectations) of the Getting Started tutorial is recommended.
```

Comment:
I think you meant to say `Step 2: Connect To Data` ? :)

Reply:
I very much did.  Thanks for the catch.

Reply:
Wait, wait -- backtracking.  It should be Step 3: Create Expectations in the tutorial.  We want them to complete the relevant tutorial steps before diving into the docs for more info, since this is the overview for step 3, we want them to have the context from completing step 3 in the tutorial before they try to wade through this.